src/main/scala/nl.knaw.dans.easy.stage/dataset/AMD.scala

```diff
@@ -52,7 +57,13 @@ object AMD extends DebugEnhancedLogging {
         <assigneeId>NOT_ASSIGNED</assigneeId>
         <wfs:workflow xmlns:wfs="http://easy.dans.knaw.nl/easy/workflow/">
           <id>dataset</id>
-          <remarks></remarks>
+          <remarks>
+            <remark>
+              <text>{ remarksContent }</text>
+              <remarkerId>{ s"easy-stage-dataset_$stageDatasetVersion" }</remarkerId>
+              <remarkDate>{ DateTime.now() }</remarkDate>
```

Comment:
```suggestion
              <remarkDate>{ DateTime.now().toString(ISODateTimeFormat.dateTime()) }</remarkDate>
```

or extract the formatter into its own value to be static over the whole project/object?